cxotime/tests/test_cxotime_linspace.py

```diff
@@ -0,0 +1,173 @@
+import astropy.units as u
+import numpy as np
+import pytest
+
+from cxotime import CxoTime
+
+
+@pytest.mark.parametrize(
+    "start, stop, dt_max, expected_len, expected_values",
+    [
+        ("2000:001", "2000:001", 1 * u.day, 2, ["2000:001", "2000:001"]),
+        (
+            "2000:001",
+            "2000:005",
+            24 * u.hour,
+            5,
+            ["2000:001", "2000:002", "2000:003", "2000:004", "2000:005"],
+        ),
+        (
+            "2000:001",
+            "2000:005",
+            2880 * u.minute,
+            3,
+            ["2000:001", "2000:003", "2000:005"],
+        ),
+        ("2000:001", "2000:005", 10 * 86400 * u.second, 2, ["2000:001", "2000:005"]),
+    ],
+)
+def test_linspace_step(start, stop, dt_max, expected_len, expected_values):
+    result = CxoTime.linspace(start, stop, step_max=dt_max)
+
+    # Confirm that the time intervals are uniform if there is any interval
+    if len(result) > 2:
```

Comment:
The function *always* returns at least one interval and so has length >= 2. The indexing below would work for length==2, but it isn't quite the right test anyway.

The things to test are:
- All the interval durations are the same, regardless of the value.
- That one interval duration is `<= min(dt_max, stop - start)`.

Reply:
I needed some abs() on that interval duration - you can let me know if that's what you had in mind for the different positive/negative cases.